spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,39 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  describe "I am taken to '/quotes' where I see the TOTAL number of quotes that were found for that keyword" do
+    describe "As well as the keyword that's being used for the search" do
+      it "As well as a list of the top 10 quotes that were found" do
+        visit root_path
+
+        fill_in :keyword, with: 'Love'
+
+        click_on "Search for Quotes"
+
+        expect(current_path).to eq("/quotes")
+
+        expect(page).to have_content('Quotes Matching Search For:')
+
+        expect(page).to have_content('Iris Murdoch')
+        expect(page).to have_content('There is only one happiness in life, to love and be loved.')
+        expect(page).to have_content('Tags:')
+        expect(page).to have_content('George Sand')
+        expect(page).to have_content('We can only learn to love by loving.')
+        expect(page).to have_content('Tags:')
+      end
+    end
+    it 'if there was nothing found with that keyword, nothing will be displayed' do
```

Comment:
This is a nice sad path for if nothing gets returned. In the version that I cloned, I was getting this error:
```
Landing Page I am taken to '/quotes' where I see the TOTAL number of quotes that were found for that keyword if there was nothing found with that keyword, nothing will be displayed
     Failure/Error: expect(page).to_not have_content('Kaelin Sleevi')
       expected not to find text "Kaelin Sleevi" in "Quotes Matching Search For: Kaelin Sleevi"
     # ./spec/features/Welcome/index_spec.rb:45:in `block (3 levels) in <top (required)>'
```
This error, in my version, is because you would still see `Kaelin Sleevi` on the page, there would just be no results returned. 

I would also probably add a test to show that you should only be getting the Top 10 (so only 10 results displayed).